scripts/tests.sh

```diff
@@ -8,6 +8,7 @@ test_dir=$(pwd)/tests
 
 pip install wheel
 pip install -r tests/requirements.txt
+export PYTHONPATH=.
```

Comment:
why did you add this?

Reply:
tests would not find `jinahub` otherwise

Reply:
I think is much safer to add `try` `catch` to import from local and then from jinahub

Reply:
It would fail anyway.

Reply:
Tried removing it and the test failed.